python/lsst/ip/diffim/imagePsfMatch.py

```diff
@@ -870,6 +870,48 @@ def makeCandidateList(self, templateExposure, scienceExposure, kernelSize, candi
 
         return candidateList
 
+    def makeKernelBasisList(self, targetFwhmPix=None, referenceFwhmPix=None,
+                            basisDegGauss=None, basisSigmaGauss=None, metadata=None):
+        """Wrapper for `lsst.ip.diffim.makeKernelBasisList` to set log messages consistently.
```

Comment:
This line is too long (93 > 79 characters).  Also, I'm not sure "consistently" is the best descriptor here (i.e consistent with what?)